<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenameFilesProgressDialog</class>
 <widget class="QDialog" name="RenameFilesProgressDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rename files ...</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../application/litsoz3.qrc">
    <normaloff>:/ls3icon.png</normaloff>:/ls3icon.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string>old name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>new name</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;   &lt;/span&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;OK&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;: file renamed/moved   &lt;/span&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;NE&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;: not existing input file   &lt;/span&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;AE&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;: new file already exists   &lt;/span&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;E&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;: error   &lt;/span&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;TE&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;: target already exists&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../application/litsoz3.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>RenameFilesProgressDialog</receiver>
   <slot>cancelClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>700</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>cancelClicked()</slot>
 </slots>
</ui>
